apps/browser-extension-wallet/src/lib/translations/en.json

```
@@ -255,7 +255,7 @@
     "deleteModal.buttons.cancel": "No",
     "reviewModal": {
       "title": "Review change to {{name}}'s handle",
-      "banner.browserDescription": "This handle may no longer belong to your contact {{name}}. Please, review to proceed.",
+      "banner.browserDescription": "This handle may no longer belongs to your contact {{name}}. Please, review to proceed.",
```

```suggestion
      "banner.browserDescription": "This handle may no longer belong to your contact {{name}}. Please, review to proceed.",
```

This is a grammar error. @daniloprates can you approve this?